VCServices/VCServicesTests/PresentationServiceTests.swift

```diff
@@ -72,15 +72,15 @@ class PresentationServiceTests: XCTestCase {
     
     func testGetRequestMalformedUri() throws {
         let expec = self.expectation(description: "Fire")
-        let malformedUrl = " "
+        let malformedUrl = "///"
```

Comment:
Why add /// ?

Reply:
The test was failing randomly, and so I guess the URLComponent(string:) initializer started supporting strings with spaces in them. I updated the string to contain "\" which is not supported.